src/openrct2/ride/coaster/LayDownRollerCoaster.cpp

```diff
@@ -16492,5 +16492,5 @@ TRACK_PAINT_FUNCTION get_track_paint_function_lay_down_rc(int32_t trackType)
         case TrackElemType::FlyerHalfLoopUninvertedDown:
             return lay_down_rc_track_half_loop_uninverted_down;
     }
-    return nullptr;
+    return get_track_paint_function_corkscrew_rc(trackType);
```

Comment:
This is not related to the rest, could you revert it?

Reply:
Do you wish me to seperate the lay-down changes to a seperate pr for ya?

Reply:
Yup.